database/sql/V027__quotes_adding_quote_kind.sql

```diff
@@ -0,0 +1,6 @@
+-- Adding new column on table quotes to differentiate between quoting methods
+-- This is required as different quotes have different expirations
+CREATE TYPE QuoteKind AS ENUM ('standard', 'eip1271onchainorder', 'presignonchainorder');
+
+ALTER TABLE quotes
+ADD COLUMN quote_kind QuoteKind Not NULL DEFAULT 'standard';
```

Comment:
ubernit:
```suggestion
ADD COLUMN kind QuoteKind Not NULL DEFAULT 'standard';
```

I feel like `quotes.quote_kind` is redundant from a naming perspective. Don't feel strongly though.

Reply:
I am undecided. You are completely right that it reads weird. But we have already `quote.order_kind` and I think there is value in keeping it explicitly different: `quote.order_kind` and `quote.quote_kind`

Reply:
Works for me. As I mentioned, I don't feel strongly and the fact that we have to `kind` fields, differentiating them more explicitly as you suggested makes sense to me.